workflow.csv

```diff
@@ -1,5 +1,6 @@
 step,protocol,dependencies
-s1_Bcl2FastQ,protocols/Bcl2FastQ.sh,
-s2_Illumina2GafFastQ,protocols/Illumina2GafFastQ.sh,s1_Bcl2FastQ
-s3_Demultiplex,protocols/Demultiplex.sh,s2_Illumina2GafFastQ
-s4_UploadSampleSheet,protocols/UploadSampleSheet.sh,s3_Demultiplex
+s01_processInterop,protocols/processInterop.sh,
+s02_Bcl2FastQ,protocols/Bcl2FastQ.sh,s0_processInterop
+s03_Illumina2GafFastQ,protocols/Illumina2GafFastQ.sh,s1_Bcl2FastQ
```

Comment:
Aan het eind van de regel staat de stap waar je een afhankelijkheid op hebt: daar is de nummering in sommige gevallen nog niet aangepast van `s[1234]` -> `s0[1234]`... Bijvoorbeeld `s1_Bcl2FastQ` moet `s02_Bcl2FastQ` zijn. Dat suggereert dat het nog niet getest is, want molgenis/compute zou er over moeten struikelen...

Reply:
ik heb het aangepast. voordat ik hem weer push zal ik hem nog een keer testen.